### ISSUE TITLE
Support for x86 device architectures

### ISSUE THREAD

**Issue by mcfarljw** [ludei/webview-plus#18]
Correct me if I'm wrong, but it looks like the current webview+ only supports ARM. This is by far the most common architecture, but based on our applications statistics about 4% of our users are using an x86 device. Supporting those devices has required us generate a separate build based on Crosswalk x86 for Cordova.

Is this something that is planned or something that might be supported in the future?

**Comment by ragcsalo** [comment]
Any news? :-)

**Comment by LibelulaV** [comment]
Yes, it is supported in the new cloud :). The beta will be released this week :). And there will be a new and up to date Webview+ too.
